### PR TITLE
My feature

### PR DESCRIPTION
**Pull request by AleksandarIvezic**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,10 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
```
Comment (srizzuto) [inline index.html]: This should be deleted

File: index.html [hunk 2]
```diff
@@ -4,10 +4,11 @@
...
+    <link rel="stylesheet" href="style.css">
+    <title>Hello World!</title>
 </head>
 <body>
-    <header>
```
Comment (srizzuto) [inline index.html]: This header should be the one containing the class name

**Comment by srizzuto** [comment]
I did the review. Check it out!

File: index.html (hunk 1 above)
Comment (AleksandarIvezic) [reply index.html]: It's modified. It's 'Hello World' now

File: index.html (hunk 2 above)
Comment (AleksandarIvezic) [reply index.html]: It is. In the next commit.

**Comment by AleksandarIvezic** [comment]
Missing linters.

**Comment by srizzuto** [comment]
All good. Ready for merge!

**Pull request closed without merging**